src/util/util.ts

```diff
@@ -0,0 +1,11 @@
+export function toBoolean(value: any): boolean {
+  return value === '' ? true : !!value;
```

Comment:
The semantics of this is a bit worrisome - one typically thinks of an empty string as a falsy value, so the function name `toBoolean` is a bit misleading.